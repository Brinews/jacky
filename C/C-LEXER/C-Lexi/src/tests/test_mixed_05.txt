ABCDx16
8888x08
1ABCDEFGx16
12ABC.4
1.5 1x08 1x16
