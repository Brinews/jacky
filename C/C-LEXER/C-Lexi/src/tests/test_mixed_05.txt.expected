{ABCDx16 , ID , 1}
{8888 , NUM , 2}
{x08 , ID , 2}
{1 , NUM , 3}
{ABCDEFGx16 , ID , 3}
{12 , NUM , 4}
{ABC , ID , 4}
{ , DOT , 4}
{4 , NUM , 4}
{1.5 , REALNUM , 5}
{1x08 , BASE08NUM , 5}
{1x16 , BASE16NUM , 5}
{ , END_OF_FILE , 7}
